packages/apktool.vm/apktool.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>apktool.vm</id>
+    <version>2.6.0.20220225</version>
+    <authors>Connor Tumbleson, Ryszard Wisniewski</authors>
```

Comment:
The dependecy only has Ryszard Wisniewski as author, not sure if we should follow that as I see in GitHub that Connor Tumbleson is the current maintainer.

Reply:
leaving as is, although in the future I will copy straight from the dependency

Reply:
It doesn't hurt to perform additional due diligence and add more authors 😄